src/tests/functional/plugin/gpu/shared_tests_instances/single_layer_tests/multiclass_nms.cpp

```diff
@@ -0,0 +1,104 @@
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "single_layer_tests/multiclass_nms.hpp"
+
+#include <vector>
+
+#include "common_test_utils/test_constants.hpp"
+
+using namespace ov::test::subgraph;
+using namespace InferenceEngine;
+using namespace ngraph;
+
+/* input format #1 with 2 inputs: bboxes N, M, 4, scores N, C, M */
+const std::vector<std::vector<ov::Shape>> shapes2Inputs = {
+    {{3, 100, 4}, {3,   1, 100}},
+    {{1, 10,  4}, {1, 100, 10 }}
+};
+
+/* input format #2 with 3 inputs: bboxes C, M, 4, scores C, M, roisnum N */
+const std::vector<std::vector<ov::Shape>> shapes3Inputs = {
+    {{1, 10, 4}, {1, 10}, {1}},
+    {{1, 10, 4}, {1, 10}, {10}},
+    {{2, 100, 4}, {2, 100}, {1}},
+    {{2, 100, 4}, {2, 100}, {10}}
+};
+
+const std::vector<int32_t> nmsTopK = {-1, 20};
+const std::vector<float> iouThreshold = {0.7f};
+const std::vector<float> scoreThreshold = {0.7f};
+const std::vector<int32_t> backgroundClass = {-1, 1};
+const std::vector<int32_t> keepTopK = {-1, 30};
+const std::vector<element::Type> outType = {element::i32};
```

Comment:
Please add i64 output type also.

Reply:
There are already lot of testcases in MulticlassNMS single-layer tests (around 1500). Adding int64 will double number of testcases and execution time. So I deliberately decided to not add int64 here. Instead, there is a testsuite for int64 outputType in unit tests. Would it be ok to test int64 in unit tests only?